src/js/components/DeploymentComponent.jsx

```diff
@@ -97,16 +96,16 @@ var DeploymentComponent = React.createClass({
         </td>
         <td>
           <ul className="list-unstyled">
-            {lazy(model.currentActions).map(function (action) {
+            {model.currentActions.map(function (action) {
               return <li key={action.app}>{action.app}</li>;
-            }).value()}
+            })}
```

Comment:
It's not even that. It's where do you draw the line.
Consistency is important so allowing something like this to go through means that other engineers will think it's OK to do, and eventually causing inconsistencies. If we're going to allow logic in templates, then we allow it always or we don't.

What do you think?